src/js/options.js

```diff
@@ -42,10 +48,21 @@ document.addEventListener('DOMContentLoaded', () => {
       items.settings.theme = theme
       items.settings.animateItems = animateItems
       items.settings.addContextMenu = addContextMenu
+      items.settings.addPageAction = addPageAction
       items.settings.openNewTab = openNewTab
       chrome.storage.sync.set({
         settings: items.settings
       })
+
+      chrome.tabs.query({}, tabs => {
```

Comment:
Line 57-65 should be above line the line with the comment  `// Get updating the settings on the options page.`
It should also have a comment on what it is doing.